Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -237,6 +238,7 @@ public final class ManifestLoader: ManifestLoaderProtocol {
         fileSystem: FileSystem? = nil,
         diagnostics: DiagnosticsEngine? = nil
     ) throws -> Manifest {
+        try self.createCacheIfNeeded()
```

Comment:
I think this lazy creation of the cache is great, and an improvement overall.  Thanks!